cli/tests/testdata/npm/no_remote_dynamic_imports/main.out

```diff
@@ -0,0 +1,5 @@
+A
+error: Uncaught (in promise) TypeError: Error getting response at http://localhost:4545/npm/registry/chalk
```

Comment:
This error is not great, but I'm unsure how we could force to use `An npm specifier was requested: "chalk", but --no-remote is specified.` only for dynamic imports

Reply:
Probably it is doing `{}` instead of `{:#}` somewhere